picard/ui/mainwindow.py

```diff
@@ -381,6 +382,10 @@ def create_actions(self):
         self.browser_lookup_action.setEnabled(False)
         self.browser_lookup_action.triggered.connect(self.browser_lookup)
 
+        self.more_results_action = QtGui.QAction(_(u"Display more results"), self)
+        self.more_results_action.setStatusTip(_(u"Display more results"))
```

Comment:
Please do:
```python
title = _(u"Display more results")
self.more_results_action = QtGui.QAction(title, self)
self.more_results_action.setStatusTip(title)
```

If this is intended to have the same translatable string for action and status tip.